pubsub/rocketmq/metadata.go

```diff
@@ -80,13 +85,18 @@ func (s *rocketMQMetaData) Decode(in interface{}) error {
 	return nil
 }
 
-func parseRocketMQMetaData(metadata pubsub.Metadata) (*rocketMQMetaData, error) {
+func parseRocketMQMetaData(metadata pubsub.Metadata, logger logger.Logger) (*rocketMQMetaData, error) {
 	rMetaData := getDefaultRocketMQMetaData()
 	if metadata.Properties != nil {
 		err := rMetaData.Decode(metadata.Properties)
 		if err != nil {
 			return nil, fmt.Errorf("rocketmq configuration error: %w", err)
 		}
 	}
+
+	if rMetaData.SendTimeOut != 0 {
+		logger.Warn("pubsub.rocketmq: metadata property 'sendTimeOut' has been deprecated and is now ignored - use 'sendTimeOutSec' instead. See: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-rocketmq/")
```

Comment:
I think the waning is fine but we can still accept this rather than ignoring it.

```
if rMetaData.SendTimeOut != 0 {
	logger.Warn("pubsub.rocketmq: metadata property 'sendTimeOut' has been deprecated - use 'sendTimeOutSec' instead. See: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-rocketmq/")
	rMetaData.SendTimeOutSec = rMetaData.SendTimeOut / 1000000
}
```

Reply:
This maybe pose a problem, if both `SendTimeOut` and `SendTimeOutSec` are set, `SendTimeOutSec` will overwritte. What do you think?